controlplane/src/core/env.schema.ts

```diff
@@ -28,4 +28,6 @@ export const envVariables = z.object({
     .string()
     .transform((val) => val === 'true')
     .optional(),
+  WEBHOOK_URL: z.string().optional(),
```

Comment:
Did you ensure that no vent is sent when those variables aren't defined?

Reply:
yes.